cfgov/unprocessed/js/routes/on-demand/secondary-navigation.js

```diff
@@ -0,0 +1,9 @@
+/* ==========================================================================
+   Scripts for Secondary Navigation Organism
+   ========================================================================== */
+
+'use strict';
+
+// List of organisms used.
+var SecondaryNavigation = require( '../../organisms/SecondaryNavigation' );
+var secondaryNavigation = new SecondaryNavigation( document.body ).init();
```

Comment:
This could be scoped to the sidebar, since secondary nav won't be anywhere else. `var secondaryNavigation = new SecondaryNavigation( document.querySelector( '.content_sidebar .o-secondary-navigation' ) ).init();`